src/System.Private.CoreLib/shared/System/Collections/Generic/IAsyncEnumerable.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace System.Collections.Generic
+{
+    public interface IAsyncEnumerable<out T>
```

Comment:
>IAsyncEnumerable [](start = 21, length = 16)

I don't know what level of xml doc is expected. Consider adding (here and other public types).

Reply:
Sure, I'll add 'em.